Utils/Dataflow/030_PDFAnalyzer/xmltable.py

```diff
@@ -133,7 +138,8 @@ def row_centery(row):
     return c
 
 class Table:
-    re_month = re.compile("(january|february|march|april|may|june|july|august|september|october|november|december)")
+    re_month = re.compile("(january|february|march|april|may|june|july|august\
+                          |september|october|november|december)")

```

Comment:
1. Those `^M` in the end of lines -- can they be removed? I believe it must be `\r` from `\r\n` (CRNL), the Windows-style new line delimiter. They might not affect the program itself, but are a bit distracting while reading the code or commit diff.

2. Does the search works properly after this change?
In my it won\`t find the "august":
```
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
>>> import re
>>> m1 = re.compile("(january|february|march|april|may|june|july|august|september|october|november|december)")
>>> m2 = re.compile("(january|february|march|april|may|june|july|august\
...                 |september|october|november|december)")
>>> m2.search("Some of my text with august mentioned")
>>> m.search("Some of my text with august mentioned")
<_sre.SRE_Match object at 0x7f4fe0bd65d0>
```
You can use this variant:
```
>>> m3 = re.compile("(january|february|march|april|may|june|july|august"
...                 "|september|october|november|december)")
>>> m3.search("Some of my text with august mentioned")
<_sre.SRE_Match object at 0x7f4fe0bd6648>
```
Strings followed each other are simply concatenated in Python.